src/OAuth2Demo/Client/Controllers/RequestResource.php

```diff
@@ -34,8 +34,6 @@ public function requestResource(Application $app)
         $response = $request->send();
         $json = json_decode((string) $response->getBody(), true);
 
-        $resource_uri = sprintf('%s%saccess_token=%s', $endpoint, false === strpos($endpoint, '?') ? '?' : '&', $token);
```

Comment:
why are you removing this? This allows the user to view the resource by clicking the link, and the `access_token` is needed for this request to authenticate.